This is what i have done to my inventory so i can still use the esx_policejob menu

AgenChicken24 opened the pull request:
I gave the code in the readme so you can copy it and also I have fixed if for some reason the attachments are duplicating it's not going to happen anymore it's in the server update ammunition event. Hope you like it. Have a nice day!

AgenChicken24 committed:
Fixed Sql

AgenChicken24 committed:
Fixed sql

AgenChicken24 committed:
Added policejob support and fixed multiple attachments

AgenChicken24 committed:
removed sql

AgenChicken24 committed:
fixed sql ty to Slypher

AgenChicken24 committed:
Added a help for the policejob menu

iSentrie commented on ESX/Inventory HUD/README.md:
````diff
@@ -59,6 +59,126 @@ TriggerServerEvent('dp_inventory:updateAmmoCount', GetHashKey(weaponModel), wepI
+	end
+end
+```
+esx_policejob/client/main.lua line 1184 replace all with the code
````
You pointing a wrong lines. I suppose you have edited or different version of esx_policejob.

FOR DUMMIES.

This is what you have to replace, that whole function.
https://github.com/esx-framework/esx_policejob/blob/9ceb66a591bca41ab10786bc795ebf04e52e2877/client/main.lua#L814

AgenChicken24 replied:
yeah I guess you are right this is the server
![image](https://user-images.githubusercontent.com/64702468/102803552-13827d00-43c1-11eb-9172-fa0544d2fe21.png)
this is the function that you need to change with mine in the read me
![image](https://user-images.githubusercontent.com/64702468/102803611-28f7a700-43c1-11eb-86a2-1a7e26d33f2e.png)
and this is when the menu is opened you so it does not check if you have the weapon but the item 
![image](https://user-images.githubusercontent.com/64702468/102803684-4f1d4700-43c1-11eb-8910-6fbee33a57d5.png)

nhokhuy197 replied:
hey done all what you said and this is what i receive, something wrong at
local hasWeapon = false
for number,items in pairs(PlayerData.inventory) do
	if items.name == v.weapon then 
		hasWeapon = true
	end
end
![333](https://user-images.githubusercontent.com/49865104/103676992-598c3300-4fb4-11eb-8252-e356c8d122ad.png)

AgenChicken24 replied:
Okay then do this replace the PlayerData.inventory in the for loop with ESX.GetPlayerData().inventory so every time you open the menu it makes sure that the inventory is updated

XvenDeR commented on ESX/Inventory HUD/README.md:
````diff
@@ -59,6 +59,126 @@ TriggerServerEvent('dp_inventory:updateAmmoCount', GetHashKey(weaponModel), wepI
 ```
+
+For police job you ca do like that and when they buy it its going to have a unique weapon id.
+This is clined line 1092
````
This line is for dragging
https://github.com/esx-framework/esx_policejob/blob/9ceb66a591bca41ab10786bc795ebf04e52e2877/client/main.lua#L1092

AgenChicken24 committed:
Fixed possible error

AgenChicken24 committed:
Update main.lua

AgenChicken24 committed:
Fixed fast item

Fixed when you are trying to enter a phone number with gcphone and if you press 1 or soo it will use a fast item this should fixe it

AgenChicken24 committed:
Update main.lua

AgenChicken24 committed:
Update main.lua

AgenChicken24 committed:
Update main.lua

AgenChicken24 committed:
Update main.lua
